tests/test_marktlokation.py

```diff
@@ -21,30 +29,124 @@ def test_serialization(self):
             ),
             energierichtung=Energierichtung.EINSP,
             bilanzierungsmethode=Bilanzierungsmethode.PAUSCHAL,
-            unterbrechbar=True,  # optional attribute
             netzebene=Netzebene.NSP,
         )
         assert malo.versionstruktur == 2, "versionstruktur was not automatically set"
-        assert malo.bo_typ == "MARKTLOKATION", "boTyp was not automatically set"
+        assert malo.bo_typ == BoTyp.MARKTLOKATION, "boTyp was not automatically set"
+
+        schema = MarktlokationSchema()
 
-        json_string = malo.dumps(
-            strip_nulls=True,
-            key_transformer=jsons.KEY_TRANSFORMER_CAMELCASE,
-            jdkwargs={"ensure_ascii": False},
+        json_string = schema.dumps(malo, ensure_ascii=False)
+        json_dict = json.loads(json_string)
+
+        assert "boTyp" in json_dict, "No camel case serialization"
+        assert "marktlokationsId" in json_dict, "No camel case serialization"
+
+        deserialized_malo: Marktlokation = schema.loads(json_string)
+
+        assert deserialized_malo.marktlokations_id == malo.marktlokations_id
+        assert deserialized_malo.marktlokations_id is not malo.marktlokations_id
```

Comment:
Irgendwas übersehe ich hier. Wie kann es gleichzeitig == und is not sein?
Aber die Tests laufen ja durch.

Reply:
Ah oder heißt das, dass es nicht das selbe Objekt ist, aber den gleichen Inhalt hat?
Kann man das als Kommentar hinzufügen oder ist das so eine bekannte Prüfung?

Reply:
hatte mich damals auch kurz verwirrt. Habe einen Kommentar hinzugefügt der das erklärt.
Den Grund für die zwei Tests hast du schon richtig erkannt.
Man prüft auf den selben Inhalt, achtet aber auch darauf, dass es sich hierbei nicht um das selbe Objekt handelt.